app/scripts/migrations/template.test.js

```diff
@@ -1,13 +1,13 @@
-import migrationTemplate from './template';
+import { migrate, version } from './template';
 
 const storage = {
-  meta: {},
+  meta: { version: -1 },
```

Comment:
At first I converted this to TypeScript as well; this adjustment was made to ensure the `meta` property met the type definition. But I switched this back to JavaScript when I realized that we don't have TypeScript tests yet (at least I didn't see any, and the lint config isn't setup for it).